lib/puppet_litmus/spec_helper_acceptance.rb

```diff
@@ -73,7 +73,7 @@ def self.configure!
         set :os, family: 'windows'
         user = node_config.dig('winrm', 'user') unless node_config.dig('winrm', 'user').nil?
         pass = node_config.dig('winrm', 'password') unless node_config.dig('winrm', 'password').nil?
-        endpoint = "http://#{ENV['TARGET_HOST']}:5985/wsman"
+        endpoint = (ENV['TARGET_HOST'] =~ %r{:\d+$}) ? "http://#{ENV['TARGET_HOST']}/wsman" : "http://#{ENV['TARGET_HOST']}:5985/wsman"
```

Comment:
Rather than breaking out the old regex, let https://ruby-doc.org/stdlib-2.4.1/libdoc/uri/rdoc/URI.html  do the parsing

Reply:
Swapped to using URI to parse and manage port/endpoint.